README.md

```diff
@@ -95,3 +95,15 @@ It provides a dashboard for Kubernetes and ks-devops.
 
 ## License
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FLinuxSuRen%2Fks-devops?ref=badge_large)
+
+## Available communication channels:
+
+- [kubesphere Devops google group ](https://groups.google.com/g/kubesphere-sig-devops/)
+    
```

Comment:
I prefer to keep these items be closer to each other. Please see the differences below.

![image](https://user-images.githubusercontent.com/1450685/125712604-5b702fef-df05-44bf-811d-d83fe18f082d.png)

![image](https://user-images.githubusercontent.com/1450685/125712626-6b66c919-2812-4c85-9a5c-4d4c3c0192cb.png)

I mean we don't need the extra space here.